$OpenBSD: patch-samples_ocl_CMakeLists_txt,v 1.1 2013/10/21 22:46:28 zhuk Exp $
Install OCL examples in the same place as other source modules do.
--- samples/ocl/CMakeLists.txt.orig	Fri Jul 26 15:20:25 2013
+++ samples/ocl/CMakeLists.txt	Fri Jul 26 15:27:09 2013
@@ -4,9 +4,9 @@ SET(OPENCV_OCL_SAMPLES_REQUIRED_DEPS opencv_core openc
                                      opencv_nonfree)
 
 ocv_check_dependencies(${OPENCV_OCL_SAMPLES_REQUIRED_DEPS})
+set(project "ocl")
 
 if(BUILD_EXAMPLES AND OCV_DEPENDENCIES_FOUND)
-  set(project "ocl")
   string(TOUPPER "${project}" project_upper)
 
   project("${project}_samples")
@@ -54,7 +54,7 @@ endif()
 if (INSTALL_C_EXAMPLES AND NOT WIN32)
   file(GLOB install_list *.c *.cpp *.jpg *.png *.data makefile.* build_all.sh *.dsp *.cmd )
   install(FILES ${install_list}
-          DESTINATION share/opencv/samples/${project}
+          DESTINATION share/OpenCV/samples/${project}
           PERMISSIONS OWNER_READ GROUP_READ WORLD_READ)
 endif()
 
